### PR TITLE
Add Skill - Extract metadata from an IT ticket in json format

### PR DESCRIPTION
**Pull request by PaulaViguri**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- Added compositional skill: extraction of relevant metadata from an it ticket in json format
- Could be relevant for various tasks in IT Automation & Network industry.

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [X] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [X] The `qna.yaml` file contains at least 5 `seed_examples`
- [X] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [X] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [X] Content does not include PII or otherwise sensitive or confidential information
- [X] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: compositional_skills/extraction/it_tickets/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+task_description: Metadata identification from an IT incidence ticket and generating and output in json format
+created_by: Paula Rodriguez
```
Comment (jjasghar) [inline compositional_skills/extraction/it_tickets/qna.yaml:2]: This needs to be your GitHub username :)

File: compositional_skills/extraction/it_tickets/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      We need the following record A to be added in the HEISUSNd.es area for the date 21th of november 2024. Thanks
+
+      Type: A
+      Value: 195.223.453.9
+
+      Thank you.
+      Affected Person:
+      Full Name: Laure Itz 
+      Affected User Location: SAINT GEORGE
+      Email: laureitz@dunjdi.es
```
Comment (jjasghar) [inline compositional_skills/extraction/it_tickets/qna.yaml:89]: Can we confirm all this information is fake? This looks like a possible real email address.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please update `qna.yaml` and confirm the comments.

File: compositional_skills/extraction/it_tickets/qna.yaml (hunk 1 above)
Comment (PaulaViguri) [reply compositional_skills/extraction/it_tickets/qna.yaml:2]: Done! Thanks for the reminder :)

File: compositional_skills/extraction/it_tickets/qna.yaml (hunk 2 above)
Comment (PaulaViguri) [reply compositional_skills/extraction/it_tickets/qna.yaml:89]: Yes! It was randomly created by myself :)

File: compositional_skills/extraction/it_tickets/qna.yaml (hunk 1 above)
Comment (jjasghar) [reply compositional_skills/extraction/it_tickets/qna.yaml:2]: Ah sorry, your actual GitHub username, not your name. So `paulaviguri`

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues, and add your GitHub username.

**Comment by PaulaViguri** [comment]
Hello, added the github username in the qna.yaml file and checked it using two linting checkers, I do not see any linting issues. Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please take a look at the linting failures on the files changed, and resolve them.

**Comment by PaulaViguri** [comment]
@jjasghar thanks a lot for pointing out the issue!! I have tried solving it :)

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
@instructlab-bot generate

**Comment by jjasghar** [comment]
```json
  {"Request_type": "CREATION",
    "Record_type": "CNAME",
    "Domain_name": "neo.joseph.com",
    "Record_value": "neo.joseph.com.cdn.clouclouclouc.net"
    "Needed_date": "17-03-2024"
    }
```

Spot checking it the SDG doesn't create valid `json`. Notice the missing `,` at the end of `clouclouclouc.net"`. Please fix the `qna.yaml` with valid `json` suggestions.

**Comment by PaulaViguri** [comment]
@jjasghar Thanks a lot for pointing that out, I have updated the qna file adding the missing comma in the first json.

**Comment by jjasghar** [comment]
Please fix the linting problem(s).

**Comment by PaulaViguri** [comment]
@jjasghar I believe the linting error is due to the version in the check-yaml.py script
image

**Review by jjasghar — APPROVED** [review]

**Comment by jjasghar** [comment]
@instructlab-bot generate

**Comment by jjasghar** [comment]
@instructlab-bot generate

**Comment by jjasghar** [comment]
@nerdalert looks SDG is broken? https://github.com/instructlab/taxonomy/pull/1058/checks?check_run_id=26194928072 I'll ping you on slack too.

**Comment by nerdalert** [comment]
@instructlab-bot generate

**Comment by nerdalert** [comment]
@instructlab-bot generate

**Comment by jjasghar** [comment]
It looks like the `sdg` has some trouble making some valid json, for instance:
```json
 {
    "Request\_type": "MODIFICATION",
    "Record\_type": "SPF",
    "Domain\_name": "acmesolutionsllc.org",
    "Record\_value": "v=spf1 include:spf.protection.outlook.com -all"
    }
```
The escaping of `\_` makes the json invalid.

**Comment by jjasghar** [comment]
As a team we have discussed this, and our core model isn't designed to deal with "code" as a whole. the `json` output is very close to code, and close enough that as above it produces _broken_ json.
Thank you for this submission, but for the time being we will close this due to this falls close enough to "code" that we can't accept it.

**Pull request closed without merging**